src/test/developTest/resources/wdr/wdr_sendung_overview_rockpalast_first_year.html

```diff
@@ -0,0 +1,893 @@
+<meta name="Description" content="Alle Videos zur Sendung in der Mediathek"/>
+<meta name="Author" content="Monika Weißenfels"/>
+<meta name="DC.Rights" content="Nur WDR"/>
+<meta name="DC.Date" content="2017-10-08T09:43+02:00" />
+<meta name="Publisher" content="wdr.de" />
+<meta property="og:image" content="http://www1.wdr.de/kultur/rockpalast106~_v-ARDFotogalerie.jpg" />
+<meta itemprop="image" content="http://www1.wdr.de/kultur/rockpalast106~_v-ARDFotogalerie.jpg" />
+<meta name="twitter:card" content="summary_large_image"/>
+<meta name="twitter:site" content="@wdr"/>
+<meta property="og:url" content="http://www1.wdr.de/mediathek/video/sendungen/rockpalast/rockpalast-136.html"/>
+<meta name="robots" content="index,follow,noarchive"/>
+<link rel="apple-touch-icon" sizes="180x180" href="/resources/img/favicon/apple-touch-icon.png">
+<link rel="icon" type="image/png" href="/resources/img/favicon/favicon-32x32.png" sizes="32x32">
+<link rel="icon" type="image/png" href="/resources/img/favicon/favicon-16x16.png" sizes="16x16">
+<link rel="manifest" href="/resources/img/favicon/manifest.json">
+<link rel="shortcut icon" href="/resources/img/favicon/favicon.ico">
+<meta name="msapplication-config" content="/resources/img/favicon/browserconfig.xml">
+<meta name="theme-color" content="#ffffff">
+<link rel="alternate" type="application/rss+xml" title="RSS" href="/mediathek/video/sendungen/rockpalast/rockpalast-136.feed" />
+<link rel="stylesheet" type="text/css" href="/resources/css/merged_base.css?version=5.44.3" media="screen"/>
+<link rel="stylesheet" type="text/css" href="/resources/css/merged_print.css?version=5.44.3" media="print"/>
+<noscript>
+<link rel="stylesheet" type="text/css" href="/resources/css/merged_noScript.css?version=5.44.3" />
+</noscript>
+<script type="text/javascript">
+var cssPath = '/resources/css/merged_';
+var webappVersion = '5.44.3';
+</script>
+<script type="text/javascript" src="/resources/js/merged_scripts.js?version=5.44.3"></script>
+<link rel="stylesheet" href="https://www.wdr.de/tv/applications/mediathek/codebase/css/headlinefix_general.css " type="text/css"/>
+<link rel="stylesheet" href="/mediathek/video/sendungen/rockpalast/podcast-button_fix_100.css" type="text/css"/>
+<link rel="stylesheet" href="/mediathek/video/sendungen/rockpalast/fix_106244_css100.css" type="text/css"/>
+</head>
+<body class="wdr themen viewL" >
+<div id="header" >
+<div class="wrapper">
+<div class="section sectionA" data-ctrl-layoutable="{'id':'laySub','action':{'default':['widthBound'],'s':['disable'],'xs':['disable']}}" data-ctrl-collapsible="{'id':'collHead','action':{'default':['openEntries','disable'],'s':['closeEntries','closeEntriesOnVanish','enable'], 'xs':['closeEntries','closeEntriesOnVanish','enable']}}">
+<div data-ctrl-layoutable="{'id':'laySub','action':{'default':['widthBound'],'s':['disable'],'xs':['disable']}}" class="mnHolder">
+<div class="fixedHeader">
+<div class="mnHolder">
+<a class="hidden" name="goToHead"></a>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "name" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AS1827)